src/course/en/articles.json

```diff
@@ -37,24 +39,38 @@
             "_banks": {
                 "_isEnabled": true,
                 "_split": "2,2",
-                "_randomisation": true
+                "_randomisation": false
             },
             "_questions": {
                 "_resetType": "hard",
                 "_canShowFeedback": false,
                 "_canShowMarking": false,
                 "_canShowModelAnswer": false
             },
-            "_isResetOnRevisit": true,
+            "_isResetOnRevisit": false,
             "_attempts": "infinite"
         },
         "_trickle": {
             "_isEnabled": true,
             "_button": {
                 "_isEnabled": true,
+                "_styleBeforeCompletion": "visible",
```

Comment:
these are all default values (or comments) so don't need to be included. Maybe remove the entire `_button` object?